pages/profile/[address]/index.tsx

```diff
@@ -249,16 +243,19 @@ export default function Profile() {
             </div>
 
             <FollowerModal followerList={followerList} />
-            <FollowingModal followingList={followingList}/>
+            <FollowingModal followingList={followingList} />
           </div>
           <div className="flex-col mx-8">
             {user?.bio ? (
-              <h3 className="text-ellipsis text-sm font-light md:font-normal"> {user.bio} </h3>
+              <h3 className="text-ellipsis text-sm font-light md:font-normal">
+                {' '}
+                {user.bio}{' '}
+              </h3>
             ) : (
               <h3 className="text-ellipsis	">
```

Comment:
could you actually remove this sample copy as well, forgot to remove it last time I worked on that page.